### PR TITLE
AB2-665 Light Theme: Msg extension to change search card theme

### PR DESCRIPTION
**Pull request by jayeshsolanki93**

### CODE REVIEW

File: mozilla-release/mobile/android/chrome/content/browser.js [hunk 1]
```diff
@@ -6524,7 +6524,8 @@ var Cliqz = {
       "Privacy:Hide",
       "Privacy:SetInfo",
       "Privacy:Show",
-      "Privacy:SetBlockingPolicy"
+      "Privacy:SetBlockingPolicy",
+      "Browser:ChangeTheme"
```
Comment (spacifici) [inline mozilla-release/mobile/android/chrome/content/browser.js]: This should be called `Search:ChangeTheme`

File: mozilla-release/mobile/android/chrome/content/browser.js [hunk 2]
```diff
@@ -6892,6 +6893,11 @@ var Cliqz = {
           name: 'updateBlocking',
           message: blockingPolicy
         });
+        break;
+      case "Search:ChangeTheme":
+        const theme = data.isLightTheme ? "white" : "blue";
```
Comment (giota-cliqz) [inline mozilla-release/mobile/android/chrome/content/browser.js]: This code is already in sync with the extension I guess, right?

Comment (jayeshsolanki93) [reply mozilla-release/mobile/android/chrome/content/browser.js]: Hi, @giota-cliqz. Should be, as it seems to apply the changes.

Comment (khaled-cliqz) [inline mozilla-release/mobile/android/chrome/content/browser.js]: Please edit this line to let extension know of the selected theme at startup time:
https://github.com/ghostery/browser-android/blob/eaea5f79164c2567f8460f1baffed5d1d339da48/mozilla-release/mobile/android/chrome/content/browser.js#L6645
please append url to cards with background parameter: `?bg=${theme}`
while theme is either `white` or `blue`

Comment (jayeshsolanki93) [reply mozilla-release/mobile/android/chrome/content/browser.js]: Hey, @khaled-cliqz I did that. Please check the new commit

File: mozilla-release/mobile/android/chrome/content/browser.js [hunk 3]
```diff
@@ -6664,14 +6665,16 @@ var Cliqz = {
 
     const id = 'firefox@ghostery.com';
     const uuid = UUIDMap.get(id);
-    const path = 'cliqz/mobile-cards/cards.html';
+    const theme = Services.prefs.getBoolPref("pref.cliqz.blue.theme", true) ? "blue" : "white";
+    const path = 'cliqz/mobile-cards/cards.html' + '?bg=' + theme;
```
Comment (khaled-cliqz) [inline mozilla-release/mobile/android/chrome/content/browser.js]: Use `const path = `cliqz/mobile-cards/cards.html?bg=${theme}``

**Pull request merged**